docs/en_us/platform_api/source/courses/overview.rst

```diff
@@ -0,0 +1,64 @@
+.. _Courses API Overview:
+
+#############################
+Courses API Overview
+#############################
+
+Use the Courses API to obtain information about edX courses.
+
+.. contents::
+   :local:
+   :depth: 1
+
+*****************************************
+Courses API Version and Status
+*****************************************
+
+The Courses API is currently at version 1.0.
+
+************************************
+Courses API Resources and Endpoints
+************************************
+
+The Courses API includes the :ref:`Courses<Courses API Courses Resource>` and
+:ref:`Blocks<Courses API Blocks Resource>` resources, and supports the following
+tasks, methods, and endpoints.
+
+=================
+Courses Resource
+=================
+
+.. list-table::
+   :widths: 20 10 70
+   :header-rows: 1
+
+   * - Task
+     - Method
+     - Endpoint
+   * - :ref:`Get the details for a specified course`
+     - GET
+     - /api/courses/v1/courses/{course_key}/
+   * - :ref:`Get the list of courses visible to a user`
+     - GET
+     - /api/courses/v1/courses/
+
+
+=================
+Blocks Resource
+=================
+
+.. list-table::
+   :widths: 20 10 70
+   :header-rows: 1
+
+   * - Task
+     - Method
+     - Endpoint
+
+   * - :ref:`Get a list of the course blocks in a course`
+     - GET
+     - /api/courses/v1/blocks/{course_key}/
```

Comment:
The section that covers /api/courses/v1/blocks/ in more detail shows the course key passed in a query parameter (course_id=edX%2FDemoX%2FDemo_Course). I think it would make more sense to have the course id in the path and not in a query parameter, but it doesn't work that way.

Unless /api/courses/v1/blocks/{course_key}/ is supposed to work, it will look a little odd to show this path in the overview. A query parameter wouldn't look too good in the overview, so this would be /api/courses/v1/blocks/. I think there might be a design problem here.